docs/en/observability/observability-ai-assistant.asciidoc

```diff
@@ -126,54 +130,64 @@ To add external data to the knowledge base in {kib}:
-    }
-}
-----
+[[obs-ai-search-connectors]]
+=== Use search connectors
+
+[TIP]
+====
+{enterprise-search-ref}/connectors.html[Search connectors] differ from the {kibana-ref}/action-types.html[Stack management -> Connectors] configured during the <<obs-ai-set-up, AI Assistant setup>>.
+Search connectors are only needed when importing external data into the Knowledge base of the AI Assistant, while the stack connector to the LLM is required for the AI Assistant to work.
+====
+
+{enterprise-search-ref}/connectors.html[Connectors] allow you to index content from external sources thereby making it available for the AI Assistant. This can greatly improve the relevance of the AI Assistant’s responses. Data can be integrated from sources such as GitHub, Confluence, Google Drive, Jira, AWS S3, Microsoft Teams, Slack, and more.
+
+These connectors are managed under *Search* -> *Content* -> *Connectors* in {kib}, they are outside of the {observability} Solution, and they require an {enterprise-search-ref}/server.html[Enterprise Search] server connected to the Elastic Stack.
+
+By default, the AI Assistant queries all search connector indices. To override this behavior and customize which indices are queried, adjust the *Search connector index pattern* setting on the <<obs-ai-settings>> page. This allows precise control over which data sources are included in AI Assistant knowledge base.
+
+To create a connector and make its content available to the AI Assistant knowledge base, do the following:
+
+. In {kib} UI, go to *Search* -> *Content* -> *Connectors* and follow the instructions to create a new connector.
++
+[NOTE]
+====
+If your {kib} Space doesn't include the `Search` solution you will have to create the connector from a different space or change your space *Solution view* setting to `Classic`.
+====
++
+For example, if you create a {enterprise-search-ref}/connectors-github.html[GitHub native connector] you will have to set a `name`, attach it to a new or existing `index`, add your `personal access token` and include the `list of repositories` to synchronize.
++
+Learn more about configuring connectors on (TBD - links)
++
+. Create a pipeline and process the data with ELSER
++
+In order to create the embeddings needed by the AI Assistant (weights and tokens into a sparse vector field) we will have to create a *ML Inference Pipeline*:
++
+.. Open the previously created connector and select the *Pipelines* tab.
+.. Select *Copy and customize* button at the `Unlock your custom pipelines` box.
+.. Select *Add Inference Pipeline* button at the `Machine Learning Inference Pipelines` box.
+.. Select ELSER ML model to add the necessary embeddings to the data.
```

Comment:
```suggestion
.. Select ELSER ML model to add the necessary embeddings to the data.
```
If ELSER ML appears on the UI, make it bold. Otherwise, add the article "the".